docs/tutorials/06_torch_runtime.ipynb

```diff
@@ -0,0 +1,1327 @@
+    "----"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 61,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "id:  123\n",
+      "execution time:  0.2116398811340332\n",
+      "score:  0.85\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('id: ', score_result.job_id)\n",
+    "print('execution time: ', score_result.execution_time)\n",
+    "print('score: ', score_result.score)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. How to use hooks in training\n",
+    "Qiskit Machine Learning offers a base hook class, `HookBase`. It is a base class for a hook that is a set of callback functions used in the training process. Users can implement their own hook classes from this base class to handle complicated callback processes. This structure provides high flexibility in the callback processes.\n",
+    "Each hook can implement 6 methods, and each method is called before/after the corresponding processes during training. The way they are called is demonstrated\n",
+    "in the following snippet:\n",
+    "```\n",
+    "    hook.before_train()\n",
+    "    for epoch in range(epochs):\n",
+    "        hook.before_epoch()\n",
+    "        for batch in train_loader:\n",
+    "            hook.before_step()\n",
+    "            train_step()\n",
+    "            hook.after_step()\n",
+    "            global_step += 1\n",
+    "        hook.after_epoch()\n",
+    "        epoch += 1\n",
+    "    hook.after_train()\n",
+    "```\n",
+    "\n",
+    "In the hook methods, users can access `TorchTrainer` via `self.trainer` to access more properties about the context (e.g., model, current iteration, or config). The following snippet describes the accessible variables that are useful for a hook.\n",
+    "\n",
+    "- TorchTrainer\n",
+    "  - model: A model to be trained.\n",
+    "  - optimizer: An optimizer used for the training.\n",
+    "  - loss_func: A loss function for the training.\n",
+    "  - train_loader: A PyTorch data loader object containing a training dataset.\n",
+    "  - val_loader: A PyTorch data loader object containing a validation dataset.\n",
+    "  - max_epoch: The maximum number of training epochs.\n",
+    "  - log_period: A logging period for a train history and a validation history. By default,\n",
+    "                there will be logs every epoch (`log_period`=1).\n",
+    "  - start_epoch: An initial epoch for warm-start training. By default, 0.\n",
+    "  - epoch: The current number of epochs.\n",
+    "  - global_step: The current number of steps.\n",
+    "  - train_logger: A logger for a training history. Use `train_logger.metrics` to access a list of logs. A log for each epoch is stored as a dictionary similar to `TorchRuntimeResult.train_history`\n",
+    "  - val_logger: A logger for a validation history. Logs are stored in the same way as train_logger.\n",
```

Comment:
It would be nice if the first identifier on each line were placed in backquotes, to denote that they are code.